DMCompiler/Compiler/DMPreprocessor/DMPreprocessor.cs

```diff
@@ -206,9 +208,9 @@ public class DMPreprocessor : IEnumerable<Token> {
                     if (IncludedInterface != null) {
                         DMCompiler.Error(new CompilerError(includedFrom ?? Location.Internal, $"Attempted to include a second interface file ({filePath}) while one was already included ({IncludedInterface})"));
                         break;
-                    }
-
-                    IncludedInterface = filePath;
+                    }                    
+                    
+                    IncludedInterface = Path.GetRelativePath(topLevelDirectory, filePath);                    
```

Comment:
It would be better to keep this as `IncludedInterface = filePath;` and instead do `compiledDream.Interface = Path.GetRelativePath(Path.GetDirectoryName(outputFile), interfaceFile);` [here](https://github.com/OpenDreamProject/OpenDream/blob/00d56970e117f0ad2e64075381bf6a0db3ddcdbe/DMCompiler/DMCompiler.cs#L226)

Reply:
That is a much better place for it.